source/materials/OpticalMaterialProperties.cc

```diff
@@ -735,6 +735,76 @@ namespace opticalprops {
     return mpt;
   }
 
+  /// PolishedAl ///
+  G4MaterialPropertiesTable* PolishedAl()
+  {
+    G4MaterialPropertiesTable* mpt = new G4MaterialPropertiesTable();
+
+    // REFLECTIVITY
+    // std::vector<G4double> ENERGIES = {
+    //   optPhotMinE_, optPhotMaxE_
+    // };
+    // std::vector<G4double> REFLECTIVITY = {
+    //   1., 1.
+    // };
+
+    std::vector<G4double> ENERGIES = {
```

Comment:
Please add a comment here with a link to the source (a doi or arxiv URL preferably).